examples/tracetest-synthetic-monitoring/readme.md

```diff
@@ -0,0 +1,12 @@
+# Synthetic monitoring with Tracetest and GitHub Actions
+
+> [Read the detailed recipe for setting up OpenTelemetry Collector with Tractest in our documentation.](https://docs.tracetest.io/examples-tutorials/recipes/running-tracetest-without-a-trace-data-store)
+
+This is a simple quick start on how to configure Tracetest and GitHub Actions to emulate synthetic monitoring using your existing Tracetest tests. The idea behind this example
+is to have a way of running a set of tests using Tracetest on a schedule and notify a slack channel in case the test fails. This way, you can keep testing your application and
```

Comment:
```suggestion
is to have a way of running a set of tests using Tracetest on a schedule and notify a Slack channel in case the test fails. This way, you can keep testing your application and
```